components/dash-core-components/src/fragments/Dropdown.react.js

```diff
@@ -35,9 +35,15 @@ export default class Dropdown extends Component {
 
     UNSAFE_componentWillReceiveProps(newProps) {
         if (newProps.options !== this.props.options) {
+            const normalizedOptions = newProps.options.map(opt =>
```

Comment:
Yeah - like this! Let's do something like this for the other components instead of `inline`